Sources/FTPropertyWrappers/UserDefault.swift

```diff
@@ -1,16 +1,34 @@
 import Foundation
 
+/// Property wrapper for storing Codable values in UserDefaults store.
 @propertyWrapper
 public final class DefaultsStore<Value: Codable> {
 
+    /// User defaults store used for this property
     private let defaults: UserDefaults
 
+    /// Encoder used to encode complex types
     private let encoder: PropertyListEncoder
+
+    /// Dencoder used to decode complex types
     private let decoder: PropertyListDecoder
 
+    /// Key for encoled property, programmer is responsible for keeping this key unique in target UserDefaults
+    /// domain.
     private let key: String
+
+    /// Default valure returned by enclodes property getter in case, that property is not found in the UserDefaults
```

Comment:
I am not sure what you wanted to write here.

```suggestion
    /// Default value returned by enclosing property getter in case, that property is not found in the UserDefaults
```